api/envoy/config/cluster/aggregate/v2alpha/cluster.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+package envoy.config.cluster.aggregate.v2alpha;
+
+option java_outer_classname = "ClusterProto";
+option java_multiple_files = true;
+option java_package = "io.envoyproxy.envoy.config.cluster.aggregate.v2alpha";
+
+import "validate/validate.proto";
+
+// [#protodoc-title: Aggregate cluster configuration]
+
+// Configuration for the aggregate cluster. See the :ref:`architecture overview
+// <arch_overview_aggregate_cluster>` for more information.
+// [#extension: envoy.clusters.aggregate]
+message ClusterConfig {
+  // Load balancing clusters in aggregate cluster. Clusters are prioritized by the orderings in the
```

Comment:
maybe `Clusters are prioritized based on the order they appear in this list`?

Reply:
done